src/ua/main.cpp

```diff
@@ -184,6 +187,17 @@ void compressChunks() {
   }
 }
 
+bool is_chunk_complete(Chunk *c) {
+    bool chunk_complete = false;
+    JSON result = fileDescribe(c->fileID);
+    string partIndex = boost::lexical_cast<string>(c->index + 1); // minimum part index is 1
```

Comment:
I got this from file.cpp:184 which indicates that chunks are 1-based while the chunk index is 0-based.  (Also see chunk.cpp:470)